lessons/Projects/project-infographic/langs/en-us/pages/rubric-infographic.adoc

```diff
@@ -0,0 +1,35 @@
+= Making Infographics Rubric
+
+[cols="2,4,4,4", options="header"]
+|===
+|
+| Wow!
+| Getting There
+| Needs Improvement
+
+| Preparatory Work
+| The display or ratio statement I submitted to my teacher formed a strong foundation for the rest of my infographic project.
```

Comment:
I think all of these should skip "I submitted to my teacher". None of the others have this phrase, and it does no useful work